### PR TITLE
Dynamic highlights list block

### PR DESCRIPTION
**Pull request by RobLoweMOJ**

### CODE REVIEW

**Review by wilson1000-MoJ — APPROVED** [review]

File: src/highlights-list/index.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <?php
+
+    // Get all the html/content that has been captured in the buffer and output via return
+    $output = ob_get_contents();
+    ob_end_clean();
```
Comment (BNewing) [inline src/highlights-list/index.php:54]: I saw the other PR had `$output = html_entity_decode($output);` in it - does this need that too?

Comment (RobLoweMOJ) [reply src/highlights-list/index.php:58]: They are not lkely to put other markup in here but its worth having it in there just in case. Good spot.

Comment (RobLoweMOJ) [reply src/highlights-list/index.php:58]: Added

Comment (BNewing) [reply src/highlights-list/index.php:58]: :D

**Pull request merged**